### DISCUSSION TITLE
Doc comment on Populate the database (/developers/academy/py/starter_text_data/text_collections)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1886] [Docs/blog comments]
# Doc comment on Populate the database (/developers/academy/py/starter_text_data/text_collections)

Here, we'll perform basic operations to communicate with Weaviate using the Python client library.

https://weaviate.io/developers/academy/py/starter_text_data/text_collections

**Comment by homanmirgolbabaee** [comment]
✅
